### PR TITLE
Language part 1

### PR DESCRIPTION
**Pull request by teh**

### CODE REVIEW

File: language/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+The Nix language is lazy and pure functional with only a small number
+of simple constructs and builtin functions.
```
Comment (Keats) [inline language/README.md]: built-in no?

File: language/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+The lazy part means that the interpreter only evaluates results the
+user requested. This allows packaging 100,000 packages in a single
+expression but only evaluating a single package for installation. The
+other packages are still there but not evaluated.
```
Comment (Keats) [inline language/README.md]: i think this paragraph is not super clear

File: language/repl.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# REPL
+
+REPL stands for "read eval print loop". It's a convenient tool that
+reads a nix-expression and prints the evaluated result.
```
Comment (Keats) [inline language/repl.md]: that sentence reads like repl are a nix only thing

File: language/repl.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+Nix has such a REPL:
+
+```
+nix-env -iA nixpkgs.nix-repl
```
Comment (Keats) [inline language/repl.md:10]: what does the iA options do? that seems a bit noisy to open a repl compared to `python` or `node` for example

Comment (Keats) [inline language/repl.md:10]: nvm just saw the stuff below, it should say that the repl exists but needs to be installed by the following command

File: language/README.md (hunk 1 above)
Comment (Gautier) [inline language/README.md]: nix documentation uses built-in

File: language/repl.md (hunk 4 above)
Comment (Gautier) [inline language/repl.md:10]: let me suggest "Nix has such a REPL which can be installed with the following command:"

File: language/README.md (hunk 1 above)
Comment (teh) [inline language/README.md]: done

**Comment by Keats** [comment]
I think you can update this whole part with the content from your slides

File: language/README.md (hunk 2 above)
Comment (teh) [inline language/README.md]: rephrased

File: language/repl.md (hunk 3 above)
Comment (teh) [inline language/repl.md]: changed

File: language/repl.md (hunk 4 above)
Comment (teh) [inline language/repl.md:10]: we can't explain everything everywhere and we've already talked about the tools

**Comment by teh** [comment]
otal

**Comment by Keats** [comment]
lgtm

**Pull request merged**